.github/workflows/maven.yml

```diff
@@ -14,28 +14,37 @@ on:
   pull_request:
     branches: [ "main" ]
 
+env:
+  MAVEN_USERNAME: ${{ secrets.TOKEN_USERNAME }}
```

Comment:
maybe NEXUS_USERNAME  and NEXUS_PASSWORD?

Reply:
done